client/my-sites/sidebar/static-data/all-sites-menu.js

```diff
@@ -6,7 +6,7 @@ import { translate } from 'i18n-calypso';
  */
 /* eslint-enable jsdoc/require-param */
 
-export default function allSitesMenu() {
+export default function allSitesMenu( { showManagePlugins = false } ) {
```

Comment:
I would use the entire object as default, this way you should call the function without passing a parameter if `showManagePlugins` is `false`

```suggestion
export default function allSitesMenu( { showManagePlugins } = { showManagePlugins: false } ) {
```

```js
allSitesMenu(); // this shouldn't trigger an error
```